### PR TITLE
edit page completed

### PR DESCRIPTION
**Pull request by yukta12**

### CODE REVIEW

File: pages/profile.php [hunk 1]
```diff
@@ -1,32 +1,59 @@
 <?php
 include_once ($_SERVER['DOCUMENT_ROOT']."/mcq/models/Person.class.php");
 include_once ($_SERVER['DOCUMENT_ROOT']."/mcq/models/Users.class.php");
+include_once ($_SERVER['DOCUMENT_ROOT']."/mcq/includes/PdoConnection.class.php");
 //include_once ($_SERVER['DOCUMENT_ROOT']."/mcq/helpers/");
 include_once("../helpers/Helper.class.php");
```
Comment (harsh-kukreja) [inline pages/profile.php]: Use absolute path

Comment (yukta12) [reply pages/profile.php]: On which line?

Comment (harsh-kukreja) [reply pages/profile.php]: Line 5 were you have included helper class

Comment (harsh-kukreja) [reply pages/profile.php]: And use helper Object for the other imports

**Comment by yukta12** [comment]
Already done

**Pull request merged**